docs/changelog/85276.yaml

```
@@ -0,0 +1,6 @@
+pr: 85276
+summary: Fix license downgrade warnings
```

Since this will be in the release notes, it's worth being a little more specific.

```suggestion
summary: Fix license downgrade warnings for API Keys and Tokens
```